bullet-train.zsh-theme

```diff
@@ -13,6 +13,9 @@
 # The default configuration, that can be overwrite in your .zshrc file
 # ------------------------------------------------------------------------------
 
+# space variable to make spaces more visible and avoid bugs
+space=' '
```

Comment:
Maybe set it without space and alter process all  of them in loop and prepend/append a space?